exp-player/addon/components/exp-frame-base.js

```diff
@@ -65,10 +65,11 @@ export default Ember.Component.extend({
         });
 
         Object.keys(this.get('meta.data').properties || {}).forEach((key) => {
-            if (this[key] && this[key].isDescriptor) return;
```

Comment:
I don't have a strong inherent preference, but this change was made to reduce the number of Jshint errors, so that linting remained useful and informative.

This setting can be overridden (`curly: false`), since the preference is a matter of opinion... but on average I think there are fewer bugs when we enforce the convention globally than when we don't. 

(and using `//jshint ignore: line` to carve out an exception in this case would just be gratuitous)